commonLibrary/src/main/java/com/skytask/common/Channels.java

```diff
@@ -0,0 +1,7 @@
+package com.skytask.common;
+
+public final class Channels {
```

Comment:
Research best practices about how to make a good design on a utility class. 
Tip: Make it immutable